### PR TITLE
EDSC-1683: Added customized exception pages

### PR DESCRIPTION
**Pull request by jboukes**
Updated PR with only the new exception pages.

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -1,26 +1,87 @@
...
...Cu4ZFZCaHdE+/69Wwf/2IVVpcSysjkPtbLhdW4hgdXhcg0t6CuvTu84lS78SSecKHQWRa+sSgF3s3VMsvFVQawCXm1YVUPhpcnRrvBnP7B54JFWwcoBsDpeBFZzBYsO3SNcre786PD8lpcnsp9qZ2urFqyQIFpMZkrtY9xeAC1uWFXXiU3dV5lW1VVL4vffMNp1xCWs8CGPqohZcR4SWH6VqzytTrG20LAP7mAlwbLS0+63nEe6wraem0VvuG9iNBH+nw1YVZDS1AXOQwKrUahgFDyKE+C10XSt2+0LPHp4fvPYdO7uuVzxrnayrrAbTsUJjbhVKrugR7tdIai+tec6MWL8xEYR92wZ/KsGQAVNG7BSupdg2TxE54W4vEcQX81xOhJY9dw2JJMi5oSVGER9vSpcRT5NQ1sf379n5l0nU/n/nNfFxqCf+GqtZMy2Mamc47gVipYRUF+b2CISIfHEyt7od7GrTQOwwmQ5W62nlQJgWb2t0D0UqTdH5SC0CKyqls+QMa6UIEECIHqbeFm8ivdBT26tkRdpB3BV2mgC+VZ6KuOoiygC6hu7tqsElZDW9nGvCpnlPIzJcMFlEl6A1cuilL5AEVgVgYVM4mulWT5pDGxo5/VOnLDmlJRaBBlcSGFY0L8dQXbDDbSbb1Xu+ikEFaQsuF4HWBEZKrhUwgvBPC9zBAmsoJ8PUWp3vEm6hePG2CdKeVrNgGVDca0LwJUu3JMv6puDdhGsYmc7yaFY9cNONxu7rhQ94X7VoIImFW8aUQtYVn7ggIQXHpRt2+6YwFIHDlhXzUoq9cRFtIQ9EY/M5+9K5fW3B+HkW1t7mZZVqvpzAit+a+NbzZ+a0CcGYpF/uGE0/tBlQ/GzikCF3InxZvWyKpuHHZ8cSmAFE5qrhMKibMNF1L42Nn33RLb4G362urBKeGbmaCl9oezehCU1ElspVsQ2GONis8d6LCSeWZoIP7JjY/+PFEHKkvJVQAfAogiswEqptWVp50Rm6Mnx1F0zOf3X/QQvBN33TT4vZufPmhtJICY1El1pggpxKQiQ6o+Gn/zgJQN/HzamuWJQNdWqIrAIrHa1tpD+0OcJJEx4pf9jMl+8OlPQr9Y9ep96QmY7Uhq6ZqNiILTkHKDg7sXD2s7eSPhZjyAlpNs10wyryphvmgw3YBUwZ7xnnsAisLwUbrQu+RNP4myT3
```
Comment (mreese84) [inline public/404.html]: This text isn't right. EDSC is available, but this is the 404 page. So "404" would be more accurate and the text below should be something more like "Sorry! The page you were looking for does not exist. Click here to return to our main search page." (and make that last bit a link to the root page.

File: public/401.html [hunk 2]
```diff
@@ -1 +1,87 @@
...
...LDmlJRaBBlcSGFY0L8dQXbDDbSbb1Xu+ikEFaQsuF4HWBEZKrhUwgvBPC9zBAmsoJ8PUWp3vEm6hePG2CdKeVrNgGVDca0LwJUu3JMv6puDdhGsYmc7yaFY9cNONxu7rhQ94X7VoIImFW8aUQtYVn7ggIQXHpRt2+6YwFIHDlhXzUoq9cRFtIQ9EY/M5+9K5fW3B+HkW1t7mZZVqvpzAit+a+NbzZ+a0CcGYpF/uGE0/tBlQ/GzikCF3InxZvWyKpuHHZ8cSmAFE5qrhMKibMNF1L42Nn33RLb4G362urBKeGbmaCl9oezehCU1ElspVsQ2GONis8d6LCSeWZoIP7JjY/+PFEHKkvJVQAfAogiswEqptWVp50Rm6Mnx1F0zOf3X/QQvBN33TT4vZufPmhtJICY1El1pggpxKQiQ6o+Gn/zgJQN/HzamuWJQNdWqIrAIrHa1tpD+0OcJJEx4pf9jMl+8OlPQr9Y9ep96QmY7Uhq6ZqNiILTkHKDg7sXD2s7eSPhZjyAlpNs10wyryphvmgw3YBUwZ7xnnsAisLwUbrQu+RNP4myT3rdLgivm5Zs8fGDuTSdThauTheLluYK+oaCLVXrJNdW8nsshTUzNnRqfTETDu3rD4oUNA/HnbhxNHPIAUOVCy5aTBjg8z3WSsIpLdx8PhrPSostxHhJYXsEKLhqWnTGJ0YbkjSZCC0ITtyHR5O3IAbL5fHHJZKZ4dVGInlxRX1Eo6ud2iDD+bWlRN1dWLzxpmjEpy3KJomFtf1jTZgzTcWYoHnqpJxI6+76Le19otlFnjDOitArXTGsZiaFXi1ITSbz/803+DARWBwnWDTZNvVzefEh7eLFFN1xLwNUGagWoyoGF63YNgUVgNcvCGpUWVpe0rpqVp0VwBRdU5YJljr0E+i2XULD0hsDyUFGxMIaFXC0/nESACy5ZhJdogZClPuEjK0aT1ygiQZXnJSKwmnXj+fHk9UpwdXfwtdEloM7QeiGwqGDICvD2dZDVBWvK8Q7cHjzIrIcZJxeBFQgBEDF5w8JlbHUtmNUypx3hlZbW1JQPzjNA1SMHLLtZWngElt8FOKDjAwqZEeg9LEpdH4o++nyDclLFAnh+ccMidpj0CaTKhYA6dmNaKyH6inRLC5wW6iwBSj0Q0CoGyYF5eePOitauJJYLT/zTckC9El5dPgWYZaVae/Glf
```
Comment (mreese84) [inline public/401.html]: This text isn't right, the app is available... but access is denied. So this should read "401 - Access Denied"

File: public/404.html (hunk 1 above)
Comment (mreese84) [reply public/404.html]: Actually... make the h1 "404 - File Not Found"

File: public/403.html [hunk 3]
```diff
@@ -1 +1,87 @@
...
...OnLDmlJRaBBlcSGFY0L8dQXbDDbSbb1Xu+ikEFaQsuF4HWBEZKrhUwgvBPC9zBAmsoJ8PUWp3vEm6hePG2CdKeVrNgGVDca0LwJUu3JMv6puDdhGsYmc7yaFY9cNONxu7rhQ94X7VoIImFW8aUQtYVn7ggIQXHpRt2+6YwFIHDlhXzUoq9cRFtIQ9EY/M5+9K5fW3B+HkW1t7mZZVqvpzAit+a+NbzZ+a0CcGYpF/uGE0/tBlQ/GzikCF3InxZvWyKpuHHZ8cSmAFE5qrhMKibMNF1L42Nn33RLb4G362urBKeGbmaCl9oezehCU1ElspVsQ2GONis8d6LCSeWZoIP7JjY/+PFEHKkvJVQAfAogiswEqptWVp50Rm6Mnx1F0zOf3X/QQvBN33TT4vZufPmhtJICY1El1pggpxKQiQ6o+Gn/zgJQN/HzamuWJQNdWqIrAIrHa1tpD+0OcJJEx4pf9jMl+8OlPQr9Y9ep96QmY7Uhq6ZqNiILTkHKDg7sXD2s7eSPhZjyAlpNs10wyryphvmgw3YBUwZ7xnnsAisLwUbrQu+RNP4myT3rdLgivm5Zs8fGDuTSdThauTheLluYK+oaCLVXrJNdW8nsshTUzNnRqfTETDu3rD4oUNA/HnbhxNHPIAUOVCy5aTBjg8z3WSsIpLdx8PhrPSostxHhJYXsEKLhqWnTGJ0YbkjSZCC0ITtyHR5O3IAbL5fHHJZKZ4dVGInlxRX1Eo6ud2iDD+bWlRN1dWLzxpmjEpy3KJomFtf1jTZgzTcWYoHnqpJxI6+76Le19otlFnjDOitArXTGsZiaFXi1ITSbz/803+DARWBwnWDTZNvVzefEh7eLFFN1xLwNUGagWoyoGF63YNgUVgNcvCGpUWVpe0rpqVp0VwBRdU5YJljr0E+i2XULD0hsDyUFGxMIaFXC0/nESACy5ZhJdogZClPuEjK0aT1ygiQZXnJSKwmnXj+fHk9UpwdXfwtdEloM7QeiGwqGDICvD2dZDVBWvK8Q7cHjzIrIcZJxeBFQgBEDF5w8JlbHUtmNUypx3hlZbW1JQPzjNA1SMHLLtZWngElt8FOKDjAwqZEeg9LEpdH4o++nyDclLFAnh+ccMidpj0CaTKhYA6dmNaKyH6inRLC5wW6iwBSj0Q0CoGyYF5eePOitauJJYLT/zTckC9El5dPgWYZaVae/G
```
Comment (mreese84) [inline public/403.html]: For consistency, this should be "403 - Forbidden"

File: public/404.html (hunk 1 above)
Comment (jboukes) [reply public/404.html]: @mreese84 agreed - I might have been following the best practices guide a little too literal here.  Update coming soon.

File: public/422.html [hunk 4]
```diff
@@ -1,26 +1,87 @@
...
...f/2IVVpcSysjkPtbLhdW4hgdXhcg0t6CuvTu84lS78SSecKHQWRa+sSgF3s3VMsvFVQawCXm1YVUPhpcnRrvBnP7B54JFWwcoBsDpeBFZzBYsO3SNcre786PD8lpcnsp9qZ2urFqyQIFpMZkrtY9xeAC1uWFXXiU3dV5lW1VVL4vffMNp1xCWs8CGPqohZcR4SWH6VqzytTrG20LAP7mAlwbLS0+63nEe6wraem0VvuG9iNBH+nw1YVZDS1AXOQwKrUahgFDyKE+C10XSt2+0LPHp4fvPYdO7uuVzxrnayrrAbTsUJjbhVKrugR7tdIai+tec6MWL8xEYR92wZ/KsGQAVNG7BSupdg2TxE54W4vEcQX81xOhJY9dw2JJMi5oSVGER9vSpcRT5NQ1sf379n5l0nU/n/nNfFxqCf+GqtZMy2Mamc47gVipYRUF+b2CISIfHEyt7od7GrTQOwwmQ5W62nlQJgWb2t0D0UqTdH5SC0CKyqls+QMa6UIEECIHqbeFm8ivdBT26tkRdpB3BV2mgC+VZ6KuOoiygC6hu7tqsElZDW9nGvCpnlPIzJcMFlEl6A1cuilL5AEVgVgYVM4mulWT5pDGxo5/VOnLDmlJRaBBlcSGFY0L8dQXbDDbSbb1Xu+ikEFaQsuF4HWBEZKrhUwgvBPC9zBAmsoJ8PUWp3vEm6hePG2CdKeVrNgGVDca0LwJUu3JMv6puDdhGsYmc7yaFY9cNONxu7rhQ94X7VoIImFW8aUQtYVn7ggIQXHpRt2+6YwFIHDlhXzUoq9cRFtIQ9EY/M5+9K5fW3B+HkW1t7mZZVqvpzAit+a+NbzZ+a0CcGYpF/uGE0/tBlQ/GzikCF3InxZvWyKpuHHZ8cSmAFE5qrhMKibMNF1L42Nn33RLb4G362urBKeGbmaCl9oezehCU1ElspVsQ2GONis8d6LCSeWZoIP7JjY/+PFEHKkvJVQAfAogiswEqptWVp50Rm6Mnx1F0zOf3X/QQvBN33TT4vZufPmhtJICY1El1pggpxKQiQ6o+Gn/zgJQN/HzamuWJQNdWqIrAIrHa1tpD+0OcJJEx4pf9jMl+8OlPQr9Y9ep96QmY7Uhq6ZqNiILTkHKDg7sXD2s7eSPhZjyAlpNs10wyryphvmgw3YBUwZ7xnnsAisLwUbrQu+RNP4myT3rdLgivm5Zs8fGDuTS
```
Comment (mreese84) [inline public/422.html]: And one more... "422 - Unprocessable Entity" and the text below should read "Earthdata Search was unable to process the contained instructions. Question can be directed to..."

**Pull request merged**